html/elements/link.json

```
@@ -1074,15 +1074,18 @@
               "support": {
                 "chrome": {
                   "version_added": "50",
-                  "notes": "<code>as=\"document\"</code> is unsupported. See <a href='https://crbug.com/593267'>bug 593267</a>."
+                  "partial_implementation": true,
+                  "notes": "Doesn’t support <code>as=audio</code>, <code>as=video</code>, <code>as=document</code>, <code>as=embed, <code>as= object</code>, <code>as=manifest</code>, <code>as=worker</code>, <code>as=sharedworker</code>, <code>as=audioworklet</code>, <code>as=paintworklet</code>, <code>as=report</code>, or <code>as=xslt</code>"
```

There's a bunch of suggestions here, so to summarize:

* I know this is terribly pedantic, but can we use quotes as you would in real HTML? That is, `<code>as="audio"</code>` instead of `<code>as=audio</code>`
* There's some spurious spacing (e.g., `as= object`)
* Let's close some unclosed tags

```suggestion
                  "notes": "Doesn’t support <code>as=\"audio\"</code>, <code>as=\"video\"</code>, <code>as=\"document\"</code>, <code>as=\"embed\"</code>, <code>as=\"object\"</code>, <code>as=\"manifest\"</code>, <code>as=\"worker\"</code>, <code>as=\"sharedworker\"</code>, <code>as=\"audioworklet\"</code>, <code>as=\"paintworklet\"</code>, <code>as=\"report\"</code>, or <code>as=\"xslt\"</code>."
```

---

Also, maybe I missed some more thoughtful grouping here, but I thought it might be nice to sort alphabetically:

```suggestion
                  "notes": "Doesn’t support <code>as=\"audio\"</code>, <code>as=\"audioworklet\"</code>, <code>as=\"document\"</code>, <code>as=\"embed\"</code>, <code>as=\"manifest\"</code>, <code>as=\"object\"</code>, <code>as=\"paintworklet\"</code>, <code>as=\"report\"</code>, <code>as=\"sharedworker\"</code>, <code>as=\"video\"</code>, <code>as=\"worker\"</code>, or <code>as=\"xslt\"</code>."
```